README.md

```diff
@@ -1 +1,82 @@
-# first-microverse-webapp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The TVMaze
+TVmaze is a web application that generates random movies from the tv Maze API. More information about the API can be found [here](https://www.tvmaze.com/api)
+
+![HomePage](./src/assets/home.jpg)
+![CommetPage](./src/assets/comment.jpg)
+
+## Built With
+
+- `HTML` ,`CSS` and `JavaScript`
+- Library : `Webpack`
+- Technologies used: `GitHub`
+- [tv-Maze API](https://www.tvmaze.com/api)                                                                                       
+- [involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270)                                                                                                                         
+
+## Live Demo
+Live site URL to Webpack basic setup
+[Live Demo Link](https://samlorlah.github.io/first-microverse-webapp/dist)
```

Comment:
- [ ] Kindly ensure you provide the correct link `/dist` results on the 404 page.